Source/Scene/Cesium3DTileGroundPolylines.js

```diff
@@ -0,0 +1,418 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/ComponentDatatype',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/IndexDatatype',
+        '../Core/Matrix4',
+        '../Core/TranslationRotationScale',
+        '../Renderer/Buffer',
+        '../Renderer/BufferUsage',
+        '../Renderer/DrawCommand',
+        '../Renderer/RenderState',
+        '../Renderer/ShaderProgram',
+        '../Renderer/ShaderSource',
+        '../Renderer/VertexArray',
+        '../Shaders/PolylineCommon',
+        '../Shaders/Appearances/PolylineColorAppearanceVS',
+        './BlendingState',
+        './Pass'
+    ], function(
+        Cartesian3,
+        ComponentDatatype,
+        defined,
+        destroyObject,
+        IndexDatatype,
+        Matrix4,
+        TranslationRotationScale,
+        Buffer,
+        BufferUsage,
+        DrawCommand,
+        RenderState,
+        ShaderProgram,
+        ShaderSource,
+        VertexArray,
+        PolylineCommon,
+        PolylineColorAppearanceVS,
+        BlendingState,
+        Pass
+    ) {
+    'use strict';
+
+    function Cesium3DTileGroundPolylines(options) {
+        this._positions = options.positions;
```

Comment:
Remind me, do we need to keep these in system memory?  Can we just keep indices to support dynamic styling?